### ISSUE TITLE
Warning when updating from version 1.0.0 to 1.0.2

### ISSUE THREAD

**Issue by xhs345** [pagekit/pagekit#616]
## Problem
Got an error message updating from version 1.0.0 to 1.0.2:

"Extracting files...
Warning: ZipArchive::extractTo(C:\xampp\htdocs\pagekit/app\system/app.php): failed to open stream: Invalid argument".

See [screenshot ](http://imgur.com/dC24ICD)

I would assume the function doesn't like the mix of directory separators.

Even though the progress bar says 100% the update is not applied

## Technical Details

- Pagekit version: 1.0.2
- OS: Windows Server 2012
- Webserver: XAMPP with Apache 2.4
- Database: MariaDB
- PHP Version: PHP 5.6.19

## Troubleshooting

- [x] I have enabled debug mode: https://pagekit.com/docs/troubleshooting/debug-mode
- [x] I have verified the server requirements: https://pagekit.com/docs/getting-started/requirements
- [x] I have tried disabling all installed extensions
- [x] I have checked the browser developer console for JavaScript errors

**Comment by MalteScharenberg** [comment]
I don't think that the mixed directory separators are a problem, since PHP support both for Windows. Furthermore `ZipArchive::extractTo(C:\xampp\htdocs\pagekit/app\system/app.php)` is an internal PHP call, which means that the path concatenation was not done by Pagekit.

Unfortunately I do not have a test environment with Windows Server 2012 at hand and Windows 8.1 works well. Therefor it would be great if you could provide some debug informations. Especially the arguments of the extract function (https://github.com/pagekit/pagekit/blob/develop/app/installer/src/SelfUpdater.php#L101) are from interest.

**Comment by xhs345** [comment]
Ok, I did a lot more testing today. Error occurred on Windows Server 2008 as well, but updating works fine on my local machine (Windows 7), all with same dev environment.

I tried extracting one file each and it looks like the update always fails on extraction of this one file ('app\system/app.php'). When I excluded it from the update the error occurred for some composer file. 
When I tried extracting the archive into the /tmp folder all the files were extracted without error.

Maybe the file is in use and can't be overwritten? Anyway, I'll stick with manual extracting the archive for now, not a big deal.
